src/modules/dashboard/components/ColonyHome/ColonyUnclaimedTransfers/ColonyUnclaimedTransfers.tsx

```diff
@@ -0,0 +1,169 @@
+  });
+
+  const { networkId, ethereal, username } = useLoggedInUser();
+  const hasRegisteredProfile = !!username && !ethereal;
+  const isNetworkAllowed = checkIfNetworkIsAllowed(networkId);
+
+  const firstItem = data?.processedColony.unclaimedTransfers[0];
+
+  const { data: tokenData } = useTokenQuery({
+    variables: { address: firstItem?.token || '' },
+  });
+
+  const transform = useCallback(
+    mergePayload({ colonyAddress, tokenAddress: firstItem?.token || '' }),
+    [colonyAddress, firstItem],
+  );
+
+  if (error) console.warn(error);
+
+  if (loading) {
+    return (
+      <MiniSpinnerLoader
+        className={styles.main}
+        loadingText={MSG.loadingData}
+      />
+    );
+  }
+
+  const token = tokenData?.token;
+
+  return data && data.processedColony.unclaimedTransfers.length ? (
+    <div className={styles.main}>
+      <Heading appearance={{ size: 'normal', weight: 'bold' }}>
+        <NavLink to={`/colony/${colonyName}/funds`}>
+          <FormattedMessage {...MSG.title} />
+        </NavLink>
+      </Heading>
+      {data && (
+        <ul>
+          <li className={styles.firstLineContainer}>
+            <div className={styles.tokenItem}>
+              <span className={styles.tokenValue}>
+                <Numeral
+                  unit={getTokenDecimalsWithFallback(token?.decimals)}
+                  value={firstItem?.amount || ''}
+                />
+              </span>
+              <span className={styles.tokenSymbol}>
+                {token?.symbol ? (
+                  <span>{token?.symbol}</span>
+                ) : (
+                  <FormattedMessage {...MSG.unknownToken} />
+                )}
+              </span>
+            </div>
+            <Tooltip
+              appearance={{ theme: 'dark', size: 'medium' }}
+              trigger="hover"
+              content={
+                <div className={styles.tooltip}>
+                  <FormattedMessage {...MSG.tooltip} />
+                </div>
+              }
+              /*
+                Not showing arrow here.
+                If the screen is narrow the tooltip gets moved and the arrow looks wacky.
+              */
+              showArrow={false}
+              placement="top-start"
+              popperProps={{
+                modifiers: [
+                  {
+                    name: 'offset',
+                    options: {
+                      offset: [5, 8],
+                    },
+                  },
+                ],
+              }}
+            >
+              <ActionButton
+                text={MSG.claimButton}
+                className={styles.button}
+                submit={ActionTypes.COLONY_CLAIM_TOKEN}
+                error={ActionTypes.COLONY_CLAIM_TOKEN_ERROR}
+                success={ActionTypes.COLONY_CLAIM_TOKEN_SUCCESS}
+                transform={transform}
+                disabled={!isNetworkAllowed || !hasRegisteredProfile}
+              />
```

Comment:
![Screenshot from 2022-04-26 18-29-05](https://user-images.githubusercontent.com/1193222/165336932-a5a0c626-0083-4a24-a722-dece191bf2a4.png)

The claim button styles are a bit weird. Should it have a underline ?

Also the tooltip is:
- missing the proper padding and text alignment
- white color for text, I think normally we have it a bit more gray / faded out _(I could be wrong though, please check)_
- no arrow

Maybe wait with this until Sayan merges his popover refactor PR #3315 and then you'll have a easier time to update the tooltip's styles

Reply:
> Maybe wait with this until Sayan merges his popover refactor PR #3315 and then you'll have a easier time to update the tooltip's styles

Indeed, I was waiting for it